src/core/control/tools/EditSelection.cpp

```diff
@@ -878,6 +891,33 @@ auto EditSelection::getSelectionTypeForPos(double x, double y, double zoom) -> C
         return CURSOR_SELECTION_DELETE;
     }
 
+    if (xmin - (DELETE_PADDING + this->btnWidth) - BORDER_PADDING <= x &&
```

Comment:
I think the code would be easier to read if some of the common computations were extracted out, e.g.
```c++
const double leftBtnMin = xmin - (DELETE_PADDING + this->btnWidth) - BORDER_PADDING;
const double leftBtnMax = xmin - (DELETE_PADDING + this->btnWidth) + BORDER_PADDING;
// etc.
```